test/helpers.test.js

```diff
@@ -39,54 +59,115 @@ describe('Helpers', () => {
+      expect(r('17000000000000000', 5)).toBe('16999999999990000');
+      expect(r('17000000000000000', 6)).toBe('16999999999900000');
+      expect(r('17000000000000000', 7)).toBe('16999999999000000');
+      expect(r('17000000000000000', 8)).toBe('16999999990000000');
+      expect(r('17000000000000000', 9)).toBe('16999999900000000');
+      expect(r('17000000000000000', 10)).toBe('16999999000000000');
+      expect(r('17000000000000000', 11)).toBe('16999990000000000');
+      expect(r('17000000000000000', 12)).toBe('16999900000000000');
+      expect(r('17000000000000000', 13)).toBe('16999000000000000');
+      expect(r('17000000000000000', 14)).toBe('16990000000000000');
+      expect(r('17000000000000000', 15)).toBe('16900000000000000');
+      expect(r('17000000000000000', 16)).toBe('16000000000000000');
+
+      expect(r('1094000000000012345', 1)).toBe('1094000000000012344');
+      expect(r('1094000000000012345', 2)).toBe('1094000000000012335');
+      expect(r('1094000000000012345', 3)).toBe('1094000000000012245');
+      expect(r('1094000000000012345', 4)).toBe('1094000000000011345');
+      expect(r('1094000000000012345', 5)).toBe('1094000000000002345');
+      expect(r('1094000000000012345', 6)).toBe('1093999999999912345');
+      expect(r('1094000000000012345', 7)).toBe('1093999999999012345');
+      expect(r('1094000000000012345', 8)).toBe('1093999999990012345');
+      expect(r('1094000000000012345', 9)).toBe('1093999999900012345');
+      expect(r('1094000000000012345', 10)).toBe('1093999999000012345');
+      expect(r('1094000000000012345', 11)).toBe('1093999990000012345');
+      expect(r('1094000000000012345', 12)).toBe('1093999900000012345');
+      expect(r('1094000000000012345', 13)).toBe('1093999000000012345');
+      expect(r('1094000000000012345', 14)).toBe('1093990000000012345');
+      expect(r('1094000000000012345', 15)).toBe('1093900000000012345');
+      expect(r('1094000000000012345', 16)).toBe('1093000000000012345');
+      expect(r('1094000000000012345', 17)).toBe('1084000000000012345');
+      expect(r('1094000000000012345', 18)).toBe('994000000000012345');
 
-      // With custom buffer size ..
-      expect(reduceCapacity('9900000000000000000', 6)).toBe(
-        '9899999999999000000',
-      );
+      // Using default value
+      expect(r('17000000000000000')).toBe('16900000000000000');
+      expect(r('91000000000000000')).toBe('90900000000000000');
+      expect(r('9900000000000000000')).toBe('9899900000000000000');
 
-      // As large as the buffer ..
-      expect(reduceCapacity('1000000000000000')).toBe('1000000000000000');
+      // Do not do anything when value as large as the buffer
+      expect(r('10000', 5)).toBe('10000');
+      expect(r('12345', 5)).toBe('12345');
+      expect(r('17000000000000000', 17)).toBe('17000000000000000');
 
-      // Too short ..
-      expect(reduceCapacity('1000')).toBe('1000');
+      // Do not do anything when value too short ..
```

Comment:
Shouldn't these be their own tests with their own names? I think if we need comments they should have their own description. To me it also makes sense if unit tests are really small and do one specific thing